printer_mods/Salveo/ArtilleryGenius_SW_conversion/CFG/printer.cfg

```diff
@@ -0,0 +1,296 @@
+# my cfg, RPI 3B+ and MKS GEN L
+
+[include mainsail.cfg]
+
+[mcu]
+serial:dev/serial/by-id/usb-1a86_USB_Serial-if00-port0 
+pin_map: arduino
+restart_method: command
+
+[printer]
+kinematics: corexz
+max_velocity: 300
+max_accel: 4000
+max_z_velocity: 300
+max_z_accel: 1000
+
+[bed_screws]
+screw1: 40,40
+screw2: 190,40
+screw3: 190,190
+screw4: 40,190
+
+[virtual_sdcard]
+path: /home/pi/gcode_files
+
+[stepper_x]
+step_pin: ar54
+dir_pin: !ar55
+enable_pin: !ar38
+step_distance: .0125

```

Comment:
Klipper no longer uses step_distance, so you'll need to update this config. there's a reference one here: https://github.com/VoronDesign/Voron-Switchwire/pull/31/files you can look at.